jvm/ml4ir-inference/scripts/generate_model.sh

```diff
@@ -30,7 +30,7 @@ RANKING_MODEL=ml4ir/applications/ranking
 
 $EXECUTOR ${RANKING_MODEL}/pipeline.py \
     --data_dir ${RANKING_MODEL}/tests/data/csv \
-    --feature_config ${RANKING_MODEL}/tests/data/configs/feature_config.yaml \
+    --feature_config ${RANKING_MODEL}/tests/data/configs/feature_config_integration_test.yaml \
```

Comment:
@ducouloa @jakemannix I've created a separate feature config to be used for python-jvm integration test. The goal is to make sure this contains the most extensive version of the feature specification.

Reply:
Thanks for the callout